content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -9,11 +9,16 @@ weight: 50
 ---
 
 <!-- overview -->
-A network policy is a specification of how groups of {{< glossary_tooltip text="pods" term_id="pod">}} are allowed to communicate with each other and other network endpoints.
 
-NetworkPolicy resources use {{< glossary_tooltip text="labels" term_id="label">}} to select pods and define rules which specify what traffic is allowed to the selected pods.
+A Kubernetes network policy is a specification of how groups of {{< glossary_tooltip text="pods" term_id="pod">}} are allowed to communicate with endpoints.  The endpoints which a Pod's NetworkPolicy will allow traffic to, or from, is be defined by combining one or more of the following constructs:
```

Comment:
I worry that this might get confused with Endpoints / EndpointSlice. Is there a word we can use in place of endpoints?

Reply:
updated. with a few other deets, good point